test/web_socket_task_test.rb

```diff
@@ -0,0 +1,214 @@
+# frozen_string_literal: true
+
+require 'kontena-websocket-client'
+require 'json'
+
+using_task_library 'controldev'
+
+WebsocketStruct = Concurrent::MutableStruct.new :ws,
+                  :received_messages, :current_state, :has_error
+
+def websocket_connect(state)
+    Kontena::Websocket::Client.connect(@url) do |ws|
+        state.ws = ws
+        state.current_state = :open
+
+        ws.read do |message|
+            state.received_messages.append(message)
+        end
+    end
+    rescue Kontena::Websocket::CloseError => exc
```

Comment:
`rescue` should have the same indent level than the method (i.e. aligned with `def`). Isn't this warned by rubocop ?